AmplifyPlugins/Predictions/AWSPredictionsPluginIntegrationTests/IdentifyBasicIntegrationTests.swift

```diff
@@ -243,108 +143,143 @@ class IdentifyBasicIntegrationTests: XCTestCase {
-                                                        switch event {
-                                                        case .success:
-                                                            completeInvoked.fulfill()
-                                                        case .failure(let error):
-                                                            XCTFail("Failed with \(error)")
-                                                        }
+            switch event {
+            case .success(let result):
+                let data = result as? IdentifyDocumentTextResult
+                XCTAssertNotNil(data)
+                XCTAssertFalse(data!.keyValues.isEmpty)
+                XCTAssertFalse(data!.fullText.isEmpty)
+                XCTAssertFalse(data!.tables.isEmpty)
+                completeInvoked.fulfill()
+            case .failure(let error):
+                XCTFail("Failed with \(error)")
+            }
         }
 
         XCTAssertNotNil(operation)
         waitForExpectations(timeout: networkTimeout)
     }
 
+    /// Given:
+    /// - An Image with plain text and form
+    /// When:
+    /// - Image is sent to Textract
+    /// Then:
+    /// - The operation completes successfully
+    /// - fullText from returned data is not empty
+    /// - keyValues from returned data is not empty
     func testIdentifyTextForms() {
-         setupAmplify()
         let testBundle = Bundle(for: type(of: self))
-        guard let url = testBundle.url(forResource: "testImageText", withExtension: "jpg") else {
-             XCTFail("Unable to find image")
-             return
+        guard let url = testBundle.url(forResource: "testImageTextForms", withExtension: "jpg") else {
+            XCTFail("Unable to find image")
+            return
         }
         let completeInvoked = expectation(description: "Completed is invoked")
 
         let operation = Amplify.Predictions.identify(type: .detectText(.form),
                                                      image: url,
                                                      options: PredictionsIdentifyRequest.Options()) { event in
-                                                        switch event {
-                                                        case .success:
-                                                            completeInvoked.fulfill()
-                                                        case .failure(let error):
-                                                            XCTFail("Failed with \(error)")
-                                                        }
+            switch event {
+            case .success(let result):
+                let data = result as? IdentifyDocumentTextResult
+                XCTAssertNotNil(data)
+                XCTAssertFalse(data!.keyValues.isEmpty)
+                XCTAssertFalse(data!.fullText.isEmpty)
+                completeInvoked.fulfill()
```

Comment:
Since we've gone through the work to properly parse the TextResults returned from the service seems like we should do some basic validation that we are parsing it correctly?